src/core/DataFile.cpp

```diff
@@ -883,6 +883,21 @@ void DataFile::upgrade_1_1_91()
 			el.setAttribute( a, ( el.attribute( a ) == "0" ) ? "1" : "0" );
 		}
 	}
+
+	list = elementsByTagName( "arpeggiator" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// Swap elements ArpDirRandom and ArpDirDownAndUp
+		if( el.attribute( "arpdir" ).toInt() == 3 )
```

Comment:
What is the advantage to parsing an `int` in this instance over doing a string comparison?